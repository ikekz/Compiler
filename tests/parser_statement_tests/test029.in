procedure p(p: integer; e: integer);
begin
end;
begin
	p(5, 5.0);
end.