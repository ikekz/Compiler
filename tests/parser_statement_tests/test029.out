(5,7) Error: Incompatible types for arg 2: got "double" expected "integer"